test/docfx.RegressionTest/RegressionTestResult.cs

```diff
@@ -20,4 +20,12 @@ internal class RegressionTestResult
     public string? CrashMessage { get; set; }
 
     public int MoreLines { get; set; }
+
+    public override string ToString() => $"{{{nameof(Succeeded)}={Succeeded}, " +
```

Comment:
I don't quite like this manual `ToString` implementation. Does changing this to a `record` automatically gives you the desired `ToString` behavior? Or is it sufficient to dump the object out as JSON?

Reply:
@herohua @yufeih 
Thanks for your review. 
Actually in the ToString implementation, I ignore some fields (i.e. HotMethods), and customize the `Diff` field output. So changing to `record` may be not help.
But after a second thought, I guess it is not necessary add ToString() method here, it is better to put the output logic directly in Console.WriteLine() statement.

Reply:
We could change the `HotMethods` to an array and auto print it here as well. This is only for our own console logging and no need to get fancy.

Reply:
HotMethods  is printed out at https://github.com/dotnet/docfx/blob/5952ea7b299a78d5b74190ac8d83db546757b596/test/docfx.RegressionTest/RegressionTest.cs#L285